dcs/triggers.py

```diff
@@ -48,15 +50,16 @@ def load_from_dict(self, data):
                 mapping.Point(imp_zone["x"], imp_zone["y"]),
                 imp_zone["radius"],
                 imp_zone["hidden"],
-                imp_zone["name"]
+                imp_zone["name"],
+                imp_zone["color"],
+                imp_zone["properties"]
```

Comment:
As you mentioned, that you have to fix the .miz files, this isn't how it should work.
pydcs should try to load any old mission file and you shouldn't need to manually fix them.
So here as in other `load_from_dict` methods either check if `properties` is in the import dict or use `imp_zone.get("properties", {})`